### PR TITLE
Add a section about the keyboard chips event

### PR DESCRIPTION
**Pull request by Tresquel**
Added information about the event and a tutorial on how to use it

### CODE REVIEW

**Comment by Tresquel** [comment]
Also should close #37

File: en/docs/modules/misc/KeyboardChip.md [hunk 1]
```diff
@@ -152,6 +164,37 @@ Input names can be accessed by `KeyboardChip.`nameOfInput, and are listed below.
 
 ## Remarks
 
+### How to use
```
Comment (kurmachu) [inline en/docs/modules/misc/KeyboardChip.md:167]: As there's the huge list of input names between these, maybe specify how to use what?
```suggestion
### How to use the event
```

File: en/docs/modules/misc/KeyboardChip.md [hunk 2]
```diff
@@ -152,6 +164,37 @@ Input names can be accessed by `KeyboardChip.`nameOfInput, and are listed below.
...
+
+![Setting the event channel](../../../assets/docs/KeyboardChip/EventChannel.png)
+
+Now go into your CPU's code and add the following function:
+
+```lua
+function eventChannel1(sender:KeyboardChip, arg:KeyboardChipEvent)
+end
+```
+Remember that it MUST be global! Prepending the local keyword will prevent the CPU from recognizing it. Adding this kind of function will cause the CPU to automatically run it whenever the user presses or unpresses a key.
```
Comment (kurmachu) [inline en/docs/modules/misc/KeyboardChip.md:184]: Prefer markdown over capitals for legibility and semantics; _remember to not use unpress?_
```suggestion
Remember that it **must** be global! Prepending the `local` keyword will prevent the CPU from recognizing it. Adding this kind of function will cause the CPU to automatically run it whenever the user presses or releases a key.
```

File: en/docs/modules/misc/KeyboardChip.md [hunk 3]
```diff
@@ -152,6 +164,37 @@ Input names can be accessed by `KeyboardChip.`nameOfInput, and are listed below.
...
 ## Remarks
 
+### How to use
+The keyboard chip takes advantage of [CPU](.CPU.md) events, which can be hard to wrap your head around at first. The following is a brief tutorial on how to log the name of the key that's currently being pressed or released.
+
+First, place a keyboard chip and a CPU on your gadget and select the CPU with the Multitool:
+
+![Selecting the CPU](./../../../assets/docs/KeyboardChip/SelectingCPU.png)
+
+Then, select EventChannels and set the first channel to the keyboard chip.
```
Comment (kurmachu) [inline en/docs/modules/misc/KeyboardChip.md:174]: Highlight `EventChannels`
```suggestion
Then, select `EventChannels` and set the first channel to the keyboard chip.
```

**Review by kurmachu — CHANGES_REQUESTED** [review]
Here are a few comments and suggestions. You don't have to actually implement them, I'll merge when all chains are resolved.

**Comment by Tresquel** [comment]
Fixed now

**Review by kurmachu — APPROVED** [review]

**Pull request merged**